### PR TITLE
New Map: "Shopping Disaster" by winniepee0

### PR DESCRIPTION
**Pull request by Avyukt1**
Creating a new PR.

### CODE REVIEW

File: shopping_disaster/map.conf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+chests.5.to = (-582,19,463)
+chests.6.n = 12
+chests.6.from = (-576,19,444)
+chests.6.to (-567,19,463)
+chests.7.n = 18
+chests.7.from = (-631,19,444)
+chests.7.to = (-618,19,463)
+chests.8.n = 12
+chests.8.from = (-646,19,444)
+chests.8.to = (-637,19,463)
```
Comment (Avyukt1) [inline shopping_disaster/map.conf:40]: According to what I saw, these chests aren't working. I couldn't even test the map without deleting it.

**Comment by winniepee** [comment]
Planning to do it again in 2021

**Comment by Panquesito7** [comment]
> Planning to do it again in 2021

I guess you should close this PR, @Avyukt1.

**Comment by Avyukt1** [comment]
Yes, okay.

**Pull request closed without merging**